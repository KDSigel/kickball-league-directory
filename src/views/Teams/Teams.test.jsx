import { render, screen } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';

it('it should check to make sure the loading happens and there are teams',




)