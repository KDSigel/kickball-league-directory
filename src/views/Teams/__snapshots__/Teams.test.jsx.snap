// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should check to make sure Teams loads 1`] = `
<div>
  <div>
    <div>
      Kicking Ball Teams
      <ul>
        <li>
          <a
            href="/team/1"
          >
            <div>
              Bridge City Sneakers
            </div>
          </a>
        </li>
        <li>
          <a
            href="/team/21"
          >
            <div>
              Red Deer Hoppers
            </div>
          </a>
        </li>
      </ul>
      <div>
        <fieldset>
          <legend>
            Add New Team
          </legend>
          <form>
            <label
              for="teamName"
            >
              Team Name
            </label>
            <input
              aria-label="team name"
              id="teamName"
              name="name"
              type="text"
              value=""
            />
            <label
              for="city"
            >
              City
            </label>
            <input
              aria-label="city"
              id="city"
              name="city"
              type="text"
              value=""
            />
            <label
              for="state"
            >
              State
            </label>
            <input
              aria-label="state"
              id="state"
              name="state"
              type="text"
              value=""
            />
            <button
              aria-label="Add New Team"
              type="submit"
            >
              Add New Team
            </button>
          </form>
        </fieldset>
      </div>
    </div>
  </div>
</div>
`;
